Sources/Recycling/ListViews/FavoriteFolders/Subviews/FavoriteFolderSelectableViewCell.swift

```diff
@@ -5,21 +5,24 @@
 import UIKit
 
 public class FavoriteFolderSelectableViewCell: RemoteImageTableViewCell {
-
     // MARK: - Private properties
 
     private let titleLabelDefaultFont: UIFont = .body
     private let titleLabelSelectedFont: UIFont = .bodyStrong
+    private var previousSeparatorInset: CGFloat = 0
 
-    private lazy var checkmarkImageView: UIImageView = {
-        let imageView = UIImageView(withAutoLayout: true)
-        imageView.layer.cornerRadius = 8
-        imageView.backgroundColor = .primaryBlue
-        imageView.image = UIImage(named: .check)
-        imageView.tintColor = .milk
-        imageView.contentMode = .scaleAspectFit
-        imageView.isHidden = true
-        return imageView
+    private lazy var rightCheckmarkView = CheckmarkView(withAutoLayout: true)
+
+    private lazy var leftCheckmarkView: CheckmarkView = {
+        let view = CheckmarkView(withAutoLayout: true)
+        view.isBordered = true
+        return view
+    }()
+
+    private lazy var editModeView: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = .white
```

Comment:
This should probably be `.milk` so it's consistent with what we set in `setSelected(_:)` and `setHighlighted(_:)`